backend/django/core/views/api_admin.py

```diff
@@ -30,7 +30,7 @@ def label_distribution(request, project_pk):
 
     Args:
         request: The POST request
-        project_pk: Primary key of the project
+        project_pk: Primary key o the project
```

Comment:
I think there might be a typo here? Was this accidental?

Reply:
I was feeling old fashioned :). No it was a typo I'll fix it.